src/coreclr/scripts/superpmi-asmdiffs.proj

```diff
@@ -55,7 +55,8 @@
     <SPMI_Partition Include="win-x64" Platform="windows" Architecture="x64" />
     <SPMI_Partition Include="win-arm64" Platform="windows" Architecture="arm64" />
     <SPMI_Partition Include="unix-x64" Platform="Linux" Architecture="x64" />
```

Comment:
should it be `linux-x64`?

Reply:
I think in this case we mean `unix` (i.e. both `osx` and `linux`) since the calling conventions on `x64` are identical (System V)